src/main/java/com/flightstats/hub/metrics/StatsdReporter.java

```diff
@@ -36,23 +36,23 @@ public void insert(String channel, long start, ChannelType type, int items, long
 
     public void event(String title, String text, String[] tags) {
         Event event = statsDFormatter.buildCustomEvent(title, text);
-        reportWithFilteredClients(title, (statsDClient -> statsDClient.recordEvent(event, tags)));
+        reportWithFilteredClients(title, (statsDClient) -> statsDClient.recordEvent(event, tags));
```

Comment:
Stylistically, I'd usually drop the parentheses around `statsDClient` since there's only one argument so they're not necessary.